INSTALL.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+# 
+# 
+# Installation script for PyKaldi
+# 
+# Usage:
+# 	./INSTALL.sh [python] [pip] [python_library]
+# 
+# 	python, pip (optional) point to the location of the executables. Defaults to $(which ..)
+# 	python_library (optional) location of the libpython shared library. As default runs a script to find it.
+# 
+# 
+
+set -e -x 
+INSTALL_NINJA=true
+
+#######################################################################################################
+# Installation configuration
+# This determine where things are going to get installed
+#######################################################################################################
+PYKALDI_DIR="$PWD"
+TOOLS_DIR="$PYKALDI_DIR/tools/extras"
+PROTOBUF_DIR="$TOOLS_DIR/protobuf"
+# NINJA_DIR="$TOOLS_DIR/ninja"
+CLIFSRC_DIR="$TOOLS_DIR/clif"
+export KALDI_DIR="$TOOLS_DIR/kaldi"
+
+PYTHON_EXECUTABLE=$(which python)
+if [[ -n "$1" ]]; then
+	PYTHON_EXECUTABLE="$1"
+	shift
+fi
+
+PYTHON_PIP=$(which pip)
```

Comment:
Remove pip stuff. We don't need pip in this script. Also, I think we need to somehow make sure the pip on PATH matches the python we are building with. I think we should take care of that in check_dependencies.sh.